src/hdf5.cpp

```diff
@@ -736,6 +736,14 @@ void writeSolverSettingsToHDF5(Solver const& solver,
     ibuffer = static_cast<int>(solver.getReturnDataReportingMode());
     H5LTset_attribute_int(file.getId(), hdf5Location.c_str(),
                           "rdrm", &ibuffer, 1);
+    
+    ibuffer = static_cast<int>(solver.getNewtonStepSteadyStateCheck());
+    H5LTset_attribute_int(file.getId(), hdf5Location.c_str(),
+                          "newton_step_steadystate_conv_", &ibuffer, 1);
```

Comment:
Do we want to keep the trailing underscore in the attribute name? No strong opinion.

Reply:
:+1: no, probably not